lib/app_profiler.rb

```diff
@@ -62,6 +63,9 @@ module Viewer
   mattr_reader :profile_enqueue_failure, default: nil
   mattr_reader :after_process_queue, default: nil
   mattr_accessor :forward_metadata_on_upload, default: false
+  mattr_accessor :profile_sampler_enabled, default: false
+  mattr_accessor :profile_sampler_status_proc, default: nil
```

Comment:
open to suggestion for this - do not like ending it with `proc`, but kind of want to convey what it actually is.

Reply:
I will actually remove it from this version and keep everything in static config. When we decide to port Core over to use the app_profiler sampler, we will need to add functionality to fetch the config dynamically. So, it is better to add something like this then.